argoverse/evaluation/detection/utils.py

```diff
@@ -323,20 +320,28 @@ def filter_instances(
     return filtered_instances
 
 
-def rank(dts: List[ObjectLabelRecord]) -> Tuple[np.ndarray, np.ndarray]:
-    """Get the rankings for the detections, according to detector confidence.
+def rank(dts: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
+    """Rank the detections in descending order according to score (detector confidence).
```

Comment:
Should we add a test case for this in the unit tests?

Reply:
indeed we should

Reply:
Agreed. I'll add one.